src/coreclr/src/jit/flowgraph.cpp

```diff
@@ -3932,12 +3959,24 @@ bool Compiler::fgCreateGCPoll(GCPollType pollType, BasicBlock* block, Statement*
 
         //  9) Mark Poll as rarely run.
```

Comment:
I see these numbers (e.g. "9)" and "5)") are they supposed to be in an order?

Reply:
These numbers aren't useful anymore, so I will remove them.